CHANGELOG.md

```diff
@@ -9,10 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- A new option whether to write down all maximally delayed gflow or not in `gflow.gflow` (#80)
+
 ### Changed
 
+- Implemented GF(2) solver independent from `z3-solver` (#80)
+
```

Comment:
Let's mention some major interface changes here (eg added or removed arguments of methods). I can do this later too